webrtc/modules/video_coding/codecs/h264/h264.cc

```diff
@@ -30,6 +34,8 @@ extern bool IsH264CodecSupportedObjC();
 bool IsH264CodecSupported() {
 #if defined(WEBRTC_IOS)
   return IsH264CodecSupportedObjC();
+#elif defined(WEBRTC_ANDROID)
+  return true;
```

Comment:
WebRtc decided to put the check for this inside the `MediaCodecVideoEncoderFactory`. So if h264 is supported, it will return a valid pointer to `H264MediaCodecEncoder`. So at this point, I think returning true is fine, and we should just let the webrtc source do its thing.